data-protocols/dsp/dsp-negotiation/dsp-negotiation-api/build.gradle.kts

```diff
@@ -19,8 +19,10 @@ plugins {
 
 dependencies {
     api(project(":data-protocols:dsp:dsp-api-configuration"))
+    api(project(":data-protocols:dsp:dsp-core"))
+    api(project(":data-protocols:dsp:dsp-http-core"))
 
-    // TODO remove this dependency by moving type constants to another module
+    // TODO remove this dependency by moving type constants to another module https://github.com/eclipse-edc/Connector/issues/3014
     api(project(":data-protocols:dsp:dsp-negotiation:dsp-negotiation-transform"))
```

Comment:
Can't this be removed now?

Reply:
Probably, will check

Reply:
unfortunately, the controllers still have dependencies on constants in the transformer module. I already created #3014

Reply:
ok